editor/gui/editor_bottom_panel.cpp

```diff
@@ -125,6 +143,65 @@ bool EditorBottomPanel::_button_drag_hover(const Vector2 &, const Variant &, But
 	return false;
 }
 
+void EditorBottomPanel::_focus_pressed_button(Button *p_btn) {
+	if (!Variant(p_btn).get_validated_object()) {
```

Comment:
This won't work, you should use an `ObjectID` instead if you need to handle freed objects

Reply:
Thanks, how would I do that?

Reply:
You would get the `ObjectID` using `get_instance_id` and then fetch the object with `ObjectDB::get_instance`